### PR TITLE
i18n typographic quotes

### PR DESCRIPTION
**Pull request by Alkarex**
Favour typographic quotes

### CODE REVIEW

File: app/i18n/de/conf.php [hunk 1]
```diff
@@ -178,7 +178,7 @@
 			'view' => 'Ansicht',
 		),
 		'hide_read_feeds' => 'Kategorien & Feeds ohne ungelesene Artikel verstecken (funktioniert nicht mit der Einstellung „Alle Artikel zeigen“)',
-		'img_with_lazyload' => 'Verwende die "träges Laden"-Methode zum Laden von Bildern',
+		'img_with_lazyload' => 'Verwende die „träges Laden“-Methode zum Laden von Bildern',
```
Comment (aledeg) [inline app/i18n/de/conf.php:181]: This one looks weird

File: app/i18n/de/sub.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
 		'title' => 'API',	// IGNORE
 	),
 	'bookmarklet' => array(
-		'documentation' => 'Ziehen Sie diese Schaltfläche auf Ihre Lesezeichen-Symbolleiste oder klicken Sie mit der rechten Maustaste darauf und wählen Sie "Als Lesezeichen hinzufügen". Klicken Sie dann auf einer beliebigen Seite, die Sie abonnieren möchten, auf die Schaltfläche "Abonnieren".',
+		'documentation' => 'Ziehen Sie diese Schaltfläche auf Ihre Lesezeichen-Symbolleiste oder klicken Sie mit der rechten Maustaste darauf und wählen Sie „Als Lesezeichen hinzufügen“. Klicken Sie dann auf einer beliebigen Seite, die Sie abonnieren möchten, auf die Schaltfläche „Abonnieren“.',
```
Comment (aledeg) [inline app/i18n/de/sub.php:19]: Here too!

File: app/i18n/he/feedback.php [hunk 3]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'הניקוי הושלם (%d מאמרים נמחקו)',
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" has been created.',	// TODO
+		'created' => 'Tag “%s”has been created.',	// TODO
 		'name_exists' => 'Tag name already exists.',	// TODO
-		'renamed' => 'Tag "%s" has been renamed to "%s".',	// TODO
+		'renamed' => 'Tag “%s”has been renamed to “%s”.',	// TODO
```
Comment (aledeg) [inline app/i18n/he/feedback.php]: Missing space?

File: app/i18n/it/feedback.php [hunk 4]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Svecchiamento completato (%d articoli cancellati)',
 	),
 	'tag' => array(
-		'created' => 'Il tag "%s" è stato creato.',
+		'created' => 'Il Tag “%s”è stato creato.',
 		'name_exists' => 'Il nome del tag è già presente.',
-		'renamed' => 'Il tag "%s" è stato rinominato in "%s".',
+		'renamed' => 'Il Tag “%s”è stato rinominato in “%s”.',
```
Comment (aledeg) [inline app/i18n/it/feedback.php]: Missing spaces?

File: app/i18n/nl/feedback.php [hunk 5]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Opschonen klaar (%d artikelen verwijderd)',
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" is aangemaakt.',
+		'created' => 'Tag “%s”is aangemaakt.',
 		'name_exists' => 'Tagnaam bestaat al.',
-		'renamed' => 'Tag "%s" hernoemd naar "%s".',
+		'renamed' => 'Tag “%s”hernoemd naar “%s”.',
```
Comment (aledeg) [inline app/i18n/nl/feedback.php]: Missing spaces?

File: app/i18n/pt-br/feedback.php [hunk 6]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Limpeza completa (%d artigos deletados)',
 	),
 	'tag' => array(
-		'created' => 'A tag "%s" foi criada.',
+		'created' => 'A Tag “%s”foi criada.',
 		'name_exists' => 'O nome da tag já existe.',
-		'renamed' => 'A tag "%s" foi renomeada para "%s".',
+		'renamed' => 'A Tag “%s”foi renomeada para “%s”.',
```
Comment (aledeg) [inline app/i18n/pt-br/feedback.php]: Missing spaces?

File: app/i18n/ru/conf.php [hunk 7]
```diff
@@ -178,7 +178,7 @@
 			'view' => 'Вид',
 		),
 		'hide_read_feeds' => 'Скрывать категории и ленты без непрочитанных статей (не работает с «Показывать все статьи»)',
-		'img_with_lazyload' => 'Использовать режим "ленивой загрузки" для загрузки картинок',
+		'img_with_lazyload' => 'Использовать режим «ленивой загрузки» для загрузки картинок',
```
Comment (aledeg) [inline app/i18n/ru/conf.php:181]: Is there a reason why those are different?

File: app/i18n/he/feedback.php (hunk 3 above)
Comment (Alkarex) [reply app/i18n/he/feedback.php]: Well spotted, thanks

File: app/i18n/ru/conf.php (hunk 7 above)
Comment (Alkarex) [reply app/i18n/ru/conf.php:181]: I picked the same convention than a few lines higher up, but I do not know what is most usual

Comment (aledeg) [reply app/i18n/ru/conf.php:181]: But on line 117, you've used the typographic quotes.

Comment (Alkarex) [reply app/i18n/ru/conf.php:181]: Yes, I believe it is a bit like in French with a few possibilities. We could ask if we find anyone mastering Russian typography :-)

File: app/i18n/de/conf.php (hunk 1 above)
Comment (Alkarex) [reply app/i18n/de/conf.php:181]: Let's get @math-GH to tell us what is most usual in German. It is picked from a similar line a bit higher up

File: app/i18n/nl/conf.php [hunk 8]
```diff
@@ -178,7 +178,7 @@
 			'view' => 'Weergave',
 		),
 		'hide_read_feeds' => 'Categorieën en feeds zonder ongelezen artikelen verbergen (werkt niet met “Toon alle artikelen” configuratie)',
-		'img_with_lazyload' => 'Gebruik "lazy load" methode om afbeeldingen te laden',
+		'img_with_lazyload' => 'Gebruik “lazy load” methode om afbeeldingen te laden',
```
Comment (Frenzie) [inline app/i18n/nl/conf.php]: In Dutch we'd traditionally use:
```suggestion
		'img_with_lazyload' => 'Gebruik „lazy load” methode om afbeeldingen te laden',
```

Though for this particular example I'm not sure if quotation marks make all that much sense anyway:
```suggestion
		'img_with_lazyload' => 'Gebruik <em>lazy load</em> methode om afbeeldingen te laden',
```

Comment (Alkarex) [reply app/i18n/nl/conf.php]: As you prefer :-)

File: app/i18n/de/conf.php (hunk 1 above)
Comment (beerisgood) [reply app/i18n/de/conf.php:181]: " is the normal version in German. 

Apple devices uses „“ by default

Comment (math-GH) [reply app/i18n/de/conf.php:181]: @Alkarex, you are right. `„“` are the typographic correct characters. While this characters are not available on the keyboard we use the `"`, but it is typographically correct a `inch` character.

File: app/i18n/de/feedback.php [hunk 9]
```diff
@@ -35,7 +35,7 @@
 	),
 	'conf' => array(
 		'error' => 'Während der Speicherung der Konfiguration trat ein Fehler auf',
-		'query_created' => 'Abfrage "%s" ist erstellt worden.',
+		'query_created' => 'Abfrage “%s” ist erstellt worden.',
```
Comment (math-GH) [inline app/i18n/de/feedback.php]: this quotation marks are not the German style, it is English style.
German quotation marks are on the bottom line (starting quotation mark) and and the top line (closing quotation mark).

If you want to learn more: https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen (sorry, it is only available in German)

File: app/i18n/de/sub.php (hunk 2 above)
Comment (math-GH) [reply app/i18n/de/sub.php:19]: that is correct

File: app/i18n/de/feedback.php [hunk 10]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Bereinigung abgeschlossen (%d Artikel gelöscht)',
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" wurde erstellt.',
+		'created' => 'Tag “%s” wurde erstellt.',
```
Comment (math-GH) [inline app/i18n/de/feedback.php]: see comment above

File: app/i18n/de/feedback.php [hunk 11]
```diff
@@ -115,9 +115,9 @@
...
 		'name_exists' => 'Tag Name existiert bereits.',
-		'renamed' => 'Der Tag "%s" wurde umbenannt in "%s".',
+		'renamed' => 'Der Tag “%s” wurde umbenannt in “%s”.',
```
Comment (math-GH) [inline app/i18n/de/feedback.php]: see comment above

**Review by math-GH — CHANGES_REQUESTED** [review]

File: app/i18n/de/feedback.php (hunk 11 above)
Comment (Alkarex) [reply app/i18n/de/feedback.php]: Fixed everywhere

File: app/i18n/de/feedback.php (hunk 9 above)
Comment (Alkarex) [reply app/i18n/de/feedback.php]: All German files fixed accordingly

File: app/i18n/de/conf.php (hunk 1 above)
Comment (Alkarex) [reply app/i18n/de/conf.php:181]: Fixed everywhere

File: app/i18n/en-us/feedback.php [hunk 12]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Purge completed (%d articles deleted)',	// IGNORE
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" has been created.',	// IGNORE
+		'created' => 'Tag “%s as been created.',	// IGNORE
```
Comment (math-GH) [inline app/i18n/en-us/feedback.php]: closing quotation mark is lost

File: app/i18n/en/feedback.php [hunk 13]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'Purge completed (%d articles deleted)',
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" has been created.',
+		'created' => 'Tag “%s as been created.',
```
Comment (math-GH) [inline app/i18n/en/feedback.php]: closing quotation mark is lost

File: app/i18n/he/feedback.php [hunk 14]
```diff
@@ -115,9 +115,9 @@
 		'purge_completed' => 'הניקוי הושלם (%d מאמרים נמחקו)',
 	),
 	'tag' => array(
-		'created' => 'Tag "%s" has been created.',	// TODO
+		'created' => 'Tag “%s as been created.',	// TODO
```
Comment (math-GH) [inline app/i18n/he/feedback.php]: closing quotation mark is lost

File: app/i18n/nl/conf.php [hunk 15]
```diff
@@ -178,7 +178,7 @@
 			'view' => 'Weergave',
 		),
 		'hide_read_feeds' => 'Categorieën en feeds zonder ongelezen artikelen verbergen (werkt niet met “Toon alle artikelen” configuratie)',
-		'img_with_lazyload' => 'Gebruik "lazy load" methode om afbeeldingen te laden',
+		'img_with_lazyload' => 'Gebruik <em>lazy load</em> methode om afbeeldingen te laden',
```
Comment (math-GH) [inline app/i18n/nl/conf.php]: why here `<em>` and in the other languages not?

**Review by math-GH — CHANGES_REQUESTED** [review]

File: app/i18n/nl/conf.php [hunk 16]
```diff
@@ -106,15 +106,15 @@
 			'type' => 'Type',	// IGNORE
 		),
 		'get_all' => 'Toon alle artikelen',
-		'get_category' => 'Toon "%s" categorie',
+		'get_category' => 'Toon “%s” categorie',
```
Comment (Frenzie) [inline app/i18n/nl/conf.php]: Could you change the rest to „adsf”?

Comment (math-GH) [reply app/i18n/nl/conf.php]: Dutch quotation marks are the same as in German?

File: app/i18n/en-us/feedback.php (hunk 12 above)
Comment (Alkarex) [reply app/i18n/en-us/feedback.php]: 🤦

File: app/i18n/nl/conf.php (hunk 16 above)
Comment (Alkarex) [reply app/i18n/nl/conf.php]: For Dutch everywhere?

File: app/i18n/nl/conf.php (hunk 15 above)
Comment (Alkarex) [reply app/i18n/nl/conf.php]: @Frenzie's personal preference for Dutch. We could update the other languages as well, but we can also leave it up to the translators

File: app/i18n/en-us/feedback.php (hunk 12 above)
Comment (Alkarex) [reply app/i18n/en-us/feedback.php]: Fixed, thanks!

**Comment by Frenzie** [comment]
@math-GH I can't really tell where you wrote it (thanks GitHub… ;-) ), but:

> why here `<em>` and in the other languages not?

Yes, I agree that would look better in English too. On the basis of only looking at that string I assumed it was an indicator of not being Dutch, for which italicized is more typical in this kind of context. And I agree with whoever translated it (or with myself if it was me) that *lui laden* (_faul laden_ in German) doesn't really help and that maintaining it as lazy loading is better.

File: app/i18n/nl/conf.php (hunk 16 above)
Comment (Frenzie) [reply app/i18n/nl/conf.php]: @math-GH Possibly, but the random German book I grabbed to check uses „adsf“, not „adsf”.

@Alkarex Yes, indeed. :-)

File: app/i18n/en-us/feedback.php (hunk 12 above)
Comment (Frenzie) [reply app/i18n/en-us/feedback.php]: A stereotypical French accent. :laughing:

File: app/i18n/nl/conf.php (hunk 16 above)
Comment (Alkarex) [reply app/i18n/nl/conf.php]: Done

**Comment by Alkarex** [comment]
Inspired by the discussion, I have updated the French version as well https://github.com/FreshRSS/FreshRSS/pull/4714/commits/3ff7d57f8378bfbc307d4d7de37f8b0ad2877706

File: app/i18n/nl/conf.php (hunk 15 above)
Comment (Alkarex) [reply app/i18n/nl/conf.php]: Done

**Comment by Alkarex** [comment]
Merging a bit fast to avoid too many conflicts with the other translations. Let's address any remaining thing in another PR

**Comment by math-GH** [comment]
Kudos to @Alkarex to improve this little, but more professional, update :)

**Pull request merged**